tasks/cycles/cycles.js

```diff
@@ -13,24 +13,53 @@
 function calculateDiscount(redemption) {
   let discount;
 
-  //PLACE YOUR CODE HERE
-
-  return discount;
+if (redemption>=0, redemption <=350){
+  discount = ('0%');
```

Comment:
just 0, without %
discount = 0;
To check yourself, please run the test: **npm run test:cycles**